modules/standard/SysBasic.chpl

```diff
@@ -18,13 +18,9 @@
  * limitations under the License.
  */
 
-/* Types for low-level programming and C integration.
+/* Types for low-level system error integration
```

Comment:
I cannot remember - are we keeping SysBasic or is it TBD? Could you add your understanding of the next step with it to a Future Work section in the PR message (maybe with a link to some issue)

Reply:
Done.  I think the short answer is that this is on our list of modules to review still (where my hope is that it will get reviewed at the same time as SysError, as they now seem very related.